pkg/granted/entrypoint.go

```diff
@@ -1,20 +1,20 @@
 package granted
 
 import (
-	"fmt"
+	"os"
 
+	"github.com/common-fate/clio"
 	"github.com/common-fate/granted/internal/build"
 	"github.com/common-fate/granted/pkg/banners"
 	"github.com/common-fate/granted/pkg/config"
 	"github.com/common-fate/granted/pkg/debug"
 	"github.com/common-fate/granted/pkg/granted/settings"
-	"github.com/fatih/color"
 	"github.com/urfave/cli/v2"
 )
 
 func GetCliApp() *cli.App {
 	cli.VersionPrinter = func(c *cli.Context) {
-		fmt.Fprintln(color.Error, banners.WithVersion(banners.Granted()))
+		clio.Log(banners.WithVersion(banners.Granted()))
```

Comment:
Similar to my other comment on clio.Log() printing to stdout.

I would like to be able to form a log that prints most of this output to stderr, but prints the version number component `v0.9.0` to stdout so it can be piped.

Though I'm not sure if thats a very standard usecase, its also not actually required for this